src/rgw/rgw_rest_s3.cc

```diff
@@ -1643,7 +1674,10 @@ void RGWListBuckets_ObjStore_S3::send_response_data(std::span<const RGWBucketEnt
 void RGWListBuckets_ObjStore_S3::send_response_end()
 {
   if (sent_data) {
-    s->formatter->close_section();
+    s->formatter->close_section(); // Buckets
+    if (!marker.empty()) {
+      s->formatter->dump_string("ContinuationToken", marker);
```

Comment:
note that the [aws doc](https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListBuckets.html#API_ListBuckets_ResponseSyntax) says "ContinuationToken is obfuscated and is not a real bucket."

but we make no effort to obfuscate here, like we don't for ListObjectsV2